sdk/maps/Azure.Maps.Common/src/MapsSasCredentialPolicy.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Maps
+{
+    /// <summary> The MapsSasCredentialPolicy used for SAS authentication. </summary>
+    public class MapsSasCredentialPolicy : HttpPipelineSynchronousPolicy
```

Comment:
Can this be shared source to avoid introducing a public type that won't be used by customers directly?

Reply:
Hi @christothes , I made it internal class. Thanks for the reminder! :)